### PR TITLE
Issue 2 user can view entries

### PR DESCRIPTION
**Pull request by wang-morris**
https://user-images.githubusercontent.com/103090771/179662860-4fbf7bf3-a809-4577-8728-e9ab748c46cd.mp4

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -25,5 +31,100 @@ function submitForm(e) {
   };
   data.entries.unshift(submission);
   $image.src = 'images/placeholder-image-square.jpg';
+  $ul.append(renderData(submission));
```
Comment (BrettAlbright) [inline js/main.js]: small update, you want to use `prepend` here to add it to the front of the other children of the ul

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
1 tiny update!

**Comment by wang-morris** [comment]
https://user-images.githubusercontent.com/103090771/179673049-e238dc01-eb0a-4bb3-a455-2a33d64c5d43.mp4

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**